moveit_core/robot_trajectory/include/moveit/robot_trajectory/robot_trajectory.h

```diff
@@ -56,6 +56,18 @@ class RobotTrajectory
 
   RobotTrajectory(const robot_model::RobotModelConstPtr& robot_model, const robot_model::JointModelGroup* group);
 
+  RobotTrajectory(const RobotTrajectory&) = delete;
```

Comment:
please `[[deprecate]]` for now and forward it to `copy() `

Reply:
cool!

Reply:
I was thinking may be we can pass some message along with the deprecated attribute.